starboard/nplb/media_can_play_mime_and_key_system_test.cc

```diff
@@ -220,53 +239,59 @@ TEST(SbMediaCanPlayMimeAndKeySystem, Invalid) {
 }
 
 TEST(SbMediaCanPlayMimeAndKeySystem, MinimumSupport) {
-  // H.264 High Profile Level 4.2
-  SbMediaSupportType result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.64002a\"; width=1920; height=1080; "
-      "framerate=30; bitrate=20000000",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
-
-  // H.264 Main Profile Level 4.2
-  result = SbMediaCanPlayMimeAndKeySystem(
+  std::vector<const char*> params_fhd{
       "video/mp4; codecs=\"avc1.4d002a\"; width=1920; height=1080; "
-      "framerate=30;",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
-
-  result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.4d002a\"; width=0; height=0; "
-      "framerate=0; bitrate=0",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
+      "framerate=30",
+      "video/mp4; codecs=\"avc1.64002a\"; width=1920; height=1080; "
+      "framerate=30",
+      "video/webm; codecs=\"vp9\"; width=1920; height=1080; framerate=30",
+      "video/webm; codecs=\"vp09.02.41.10.01.09.16.09.00\"; width=1920; "
+      "height=1080; framerate=30",
+      "video/mp4; codecs=\"av01.0.09M.08\"; width=1920; height=1080; "
+      "framerate=30",
+      "video/mp4; codecs=\"av01.0.09M.10.0.110.09.16.09.0\"; width=1920; "
+      "height=1080; framerate=30",
+  };
 
-  result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.4d002a\"; width=-0; height=-0; "
-      "framerate=-0; bitrate=-0",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
+  std::vector<const char*> params_4k{
+      "video/webm; codecs=\"vp9\"; width=3840; height=2160; framerate=30",
+      "video/webm; codecs=\"vp09.02.51.10.01.09.16.09.00\"; width=3840; "
+      "height=2160; framerate=30",
+      "video/mp4; codecs=\"av01.0.12M.08\"; width=3840; height=2160; "
+      "framerate=30",
+      "video/mp4; codecs=\"av01.0.13M.10.0.110.09.16.09.0\"; width=3840; "
+      "height=2160; framerate=30",
+  };
 
-  // H.264 Main Profile Level 2.1
-  result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.4d0015\"; width=432; height=240; "
-      "framerate=15;",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
+  std::vector<const char*> params_8k{
+      "video/webm; codecs=\"vp9\"; width=3840; height=2160; framerate=60",
+      "video/webm; codecs=\"vp09.02.51.10.01.09.16.09.00\"; width=3840; "
+      "height=2160; framerate=60",
```

Comment:
For this one, 8k devices are supposed to support vp9 at 60fps since 8k vp9 isn't served. I ran into the same problems a few months ago, but had it cleared up.

Reply:
Ah, yes, you're right.